src/transaction.ts

```diff
@@ -1281,7 +1295,21 @@ export class Snapshot extends EventEmitter {
       ...query,
     };
     return startTrace('Snapshot.runStream', traceConfig, span => {
+      let attempt = 0;
       const makeRequest = (resumeToken?: ResumeToken): Readable => {
+        if (!resumeToken) {
```

Comment:
I believe we are missing this check in above `createReadStream` .  Keep it consistent in both methods.

Reply:
I added it, thank you!